sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/pipeline/_pipeline_job_helpers.py

```diff
@@ -4,10 +4,10 @@
 import re
 from typing import Dict, List, Tuple, Union
 
-from azure.ai.ml._restclient.v2022_10_01_preview.models import InputDeliveryMode
-from azure.ai.ml._restclient.v2022_10_01_preview.models import JobInput as RestJobInput
-from azure.ai.ml._restclient.v2022_10_01_preview.models import JobOutput as RestJobOutput
-from azure.ai.ml._restclient.v2022_10_01_preview.models import Mpi, PyTorch, TensorFlow
+from azure.ai.ml._restclient.v2022_12_01_preview.models import InputDeliveryMode
```

Comment:
We missed this in previous PR?

Reply:
Yes, but I find that these modification may not affect running test. I notice these lines and modify in this pr.